test/units/module_utils/facts/system/test_date_time.py

```diff
@@ -0,0 +1,79 @@
+# unit tests for ansible system date_time fact collectors
+# -*- coding: utf-8 -*-
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import datetime
+import time
+
+from units.compat.mock import Mock, patch
```

Comment:
You imported patch but didn't use it. It's there to be used but it's also preferable to use pytest's built-in `monkeypatch` fixture.

Reply:
I should have mentioned earlier that for net new tests, we prefer `pytest` style tests rather than `unittest` style tests.